w4af/core/ui/api/utils/scans.py

```diff
@@ -47,6 +60,13 @@ def create_temp_profile(scan_profile):
     with open(scan_profile_file, 'w') as profile_fh:
         profile_fh.write(scan_profile)
 
+    
+
+    print("***************create_temp_profile*********************")
```

Comment:
Please don't put `print` statements in production code. It's fine if you need them for debugging, but anything that should go to the user should be directed through the `OutputManager` (`om.out.debug(msg)`).